test/lint/lint-include-guards.sh

```diff
@@ -10,7 +10,16 @@ export LC_ALL=C
 HEADER_ID_PREFIX="UNITE_"
 HEADER_ID_SUFFIX="_H"
 
-REGEXP_EXCLUDE_FILES_WITH_PREFIX="src/(crypto/ctaes/|leveldb/|secp256k1/|tinyformat.h|univalue|usbdevice/)"
+REGEXP_EXCLUDE_FILES_WITH_PREFIX="src/(\
+better-enums/|\
+crypto/ctaes/|\
+key/mnemonic/|\
+leveldb/|\
+secp256k1/|\
+tinyformat.h|\
+unilib/|\
+univalue|\
+usbdevice/)"
```

Comment:
Why the exclude list is longer than before? Are all of them external libraries? (I guess yes, just to confirm)

Reply:
Precisely